pkg/scanners/helm/scanner.go

```diff
@@ -89,7 +90,49 @@ func (s *Scanner) SetPolicyFilesystem(policyFS fs.FS) {
 
 func (s *Scanner) ScanFS(ctx context.Context, target fs.FS, path string) (scan.Results, error) {
 
-	helmParser := parser.New()
+	var results []scan.Result
+	if err := fs.WalkDir(target, path, func(path string, d fs.DirEntry, err error) error {
+		select {
+		case <-ctx.Done():
+			return ctx.Err()
+		default:
+		}
+
+		if err != nil {
+			return err
+		}
+
+		if path == "." {
```

Comment:
```suggestion
		if d.IsDir() {
```
Am I right in thinking we can skip all directories here? As we only want archives or `Chart.yaml`s?

Reply:
done